src/sentry_plugins/hipchat_ac/templates/sentry_hipchat_ac/configure.html

```diff
@@ -1,5 +1,32 @@
 {% extends "sentry_hipchat_ac/configure_layout.html" %}
+
+{% load sentry_helpers %}
+
 {% block body %}
+
+<script>
+// open sentry login in another window and then receive a message of whether user is logged in. If user is logged in, refresh the page, so that user can continue through configuration flow
+
+  function openLogin() {
+    var top = screen.height / 2 - 350
+    var left = screen.width / 2 - 300
+    window.open(
+      ('{% system_origin %}' + "/auth/login?next=" + encodeURIComponent('/auth/close/')),
```

Comment:
_Technically_ no reason to need to append here anymore, but doesn't particularly matter. `'{% system_origin %}/auth/login?next='` would be just fine.